Issue#26

viktoriussuwandi opened the pull request:
issue #26

Hi @XanderRubio,
I'm just trying to jump on this issue, please check it out

viktoriussuwandi committed:
Update README.md

viktoriussuwandi committed:
Update README.md

XanderRubio commented:
Hi Viktorius! Thank you for your time into #26. I'm looking at your commits now.

XanderRubio committed:
Update README.md

edited the 'issues closed' and "PRs closed" URL for the img.shields to the Before I Die repo address

XanderRubio commented:
Great! Looks excellent with the addition, and we just needed to add the correct URL address of the Before I Die Code repo for the badges of issues closed and pull requests pulled to be linked to our repo info. 

`[![issues closed](https://img.shields.io/github/issues-closed/BeforeIDieCode/BeforeIDieAchievements)](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues)
[![PRs closed](https://img.shields.io/github/issues-pr-closed/BeforeIDieCode/BeforeIDieAchievements)](https://github.com/BeforeIDieCode/BeforeIDieAchievements/pulls)`

Thank you, Viktorius! for acting right away on this issue! Please go ahead and star ⭐️ this repository and also [add your text and image to the Before I Die website](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md) in addition, add your name to the [Contributor.md](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTORS.md) and you have contributed! Thank you again and have a great day! 

#26 is now closed ✅
